temba/settings_common.py

```diff
@@ -1423,6 +1423,10 @@ def traces_sampler(sampling_context):  # pragma: no cover
 
 GOOGLE_FONT_API_KEY = os.environ.get("GOOGLE_FONT_API_KEY", "")
 
+# Translation services config
+GOOGLE_TRANSLATE_API_KEY = os.environ.get("GOOGLE_TRANSLATE_API_KEY", "")
```

Comment:
Make those keys coming from the org config field. Nelson will provide a new mock for the org home page, but the JSON structure would be like this:
```
{
   "translator_service": {
      "provider": "google OR deepl",
      "api_key": "1234-abcd"
   }
}
```